test/toolkits/style_transfer/test_style_transfer.cxx

```diff
@@ -0,0 +1,39 @@
+/* Copyright © 2019 Apple Inc. All rights reserved.
+ *
+ * Use of this source code is governed by a BSD-3-clause license that can
+ * be found in the LICENSE.txt file or at
+ * https://opensource.org/licenses/BSD-3-Clause
+ */
+
+#define BOOST_TEST_MODULE
+
+#include <boost/test/unit_test.hpp>
+#include <core/util/test_macros.hpp>
+#include <toolkits/style_transfer/style_transfer.hpp>
+
+#include <map>
+#include <memory>
+
+#include "utils.hpp"
+
+using namespace turi::style_transfer;
+
```

Comment:
Do we want more tests here?

Reply:
Added TODOs to address the coming tests.